frontend/src/plugins/impl/DataTablePlugin.tsx

```diff
@@ -99,7 +101,10 @@ type Functions = {
   }>;
 };
 
-type S = Array<string | number>;
+// TODO: this was Array<number | string>
+// I don't quite get this. Does it represent the row-id?
+// Is the row-id the index property in our case?
+type S = Array<{ rowId: string; columnName?: string }>;
```

Comment:
yea i think `Array<number | string>` was a mistake. i think `Array<number>` is likely what is shouldve been. but what you have now looks correct

Reply:
actually, since the backend does support just an list of `int`, lets do `Array<number | { rowId: string; columnName: string }>`

Reply:
i think its a string because `Object.keys()` can weirdly return strings if i recall, so i think thats why we made it that. probably better to just handle `Array<number | string | { rowId: string; columnName: string }>`